app/services/after_block_domain_from_account_service.rb

```diff
@@ -67,4 +70,8 @@ def notify_of_severed_relationships!
   def domain_block_event
     @domain_block_event ||= RelationshipSeveranceEvent.create!(type: :user_domain_block, target_name: @domain)
   end
+
+  def notify_streaming!
+    redis.publish('system', Oj.dump(event: :domain_blocks_changed, account: @account.id, target_domain: @domain))
```

Comment:
Additionally, there is no comparable service for "after a domain has been unblocked by an account", and I'm not sure where best to implement this, I'm currently thinking in a `after_destroy` on the `AccountDomainBlock` model, since the removal of an account domain block can happen from many code paths (API, Imports, etc)